src/utils/businessHours.ts

```diff
@@ -53,29 +61,34 @@ export function calculateTimeToRespondBy(
 }
 
 export function calculateSLOViolationTriage(
-  target_name: string,
+  targetName: string,
   labels: any,
   repo: string,
   org: string
 ): string | null {
   // calculate time to triage for issues that come in with untriaged label
-  if (target_name === WAITING_FOR_PRODUCT_OWNER_LABEL) {
+  if (targetName === WAITING_FOR_PRODUCT_OWNER_LABEL) {
     const productArea = labels
       ?.find((label) => label.name.startsWith(PRODUCT_AREA_LABEL_PREFIX))
       ?.name.slice(PRODUCT_AREA_LABEL_PREFIX.length);
-    return calculateTimeToRespondBy(MAX_TRIAGE_DAYS, productArea, repo, org);
+    return calculateTimeToRespondBy({
+      numDays: MAX_TRIAGE_DAYS,
+      productArea,
+      repo,
+      org,
+    });
   }
   // calculate time to triage for issues that are rerouted
   else if (
-    target_name.startsWith(PRODUCT_AREA_LABEL_PREFIX) &&
+    targetName.startsWith(PRODUCT_AREA_LABEL_PREFIX) &&
     labels?.some((label) => label.name === WAITING_FOR_PRODUCT_OWNER_LABEL)
   ) {
-    return calculateTimeToRespondBy(
-      MAX_TRIAGE_DAYS,
-      target_name.slice(PRODUCT_AREA_LABEL_PREFIX.length),
+    return calculateTimeToRespondBy({
```

Comment:
Surprising that this line is not hit by tests? Maybe the instrumentation is being weird?